peerconnection_js.go

```diff
@@ -50,6 +50,14 @@ func (api *API) NewPeerConnection(configuration Configuration) (_ *PeerConnectio
 	}, nil
 }
 
+// NewPeerConnectionFromJS creates a new PeerConnection with the provided underlying JS RTCPeerConnectoin object.
+func NewPeerConnectionFromJS(pc js.Value) *PeerConnection {
```

Comment:
I don't know how you feel about enlarging the API surface with this, or the divergence between wasm and non-wasm APIs.

One alternative would be to make `underlying` public, maybe embedded. This allows:

```
pc := webrtc.PeerConnection{Value: jsPC}
```

(We'd also have to deal with `api` being nil.)